From 56cfe32b834094fa2aaeb3b8c852d3219be10d07 Mon Sep 17 00:00:00 2001
From: Ashwin Sekhar T K <asekhar@marvell.com>
Date: Mon, 13 Feb 2023 18:52:13 +0530
Subject: [PATCH 213/955] common/cnxk: update ROC erratas

Update the models where errata IPBUNPA-37480 is applicable.

Signed-off-by: Ashwin Sekhar T K <asekhar@marvell.com>
Change-Id: I91bf17fd90e49f2bc5182056507e8ad87389173a
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/96768
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Base-Tests: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Jerin Jacob Kollanukkaran <jerinj@marvell.com>
---
 drivers/common/cnxk/roc_errata.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/common/cnxk/roc_errata.h b/drivers/common/cnxk/roc_errata.h
index 9954b7da321c3..837f4ac6ffd00 100644
--- a/drivers/common/cnxk/roc_errata.h
+++ b/drivers/common/cnxk/roc_errata.h
@@ -25,7 +25,8 @@ roc_errata_nix_has_cq_min_size_4k(void)
 static inline bool
 roc_errata_npa_has_no_fc_stype_ststp(void)
 {
-	return roc_model_is_cn10ka_a0() ? true : false;
+	return roc_model_is_cn10ka_a0() || roc_model_is_cn10ka_a1() || roc_model_is_cnf10ka_a0() ||
+	       roc_model_is_cnf10kb_a0();
 }
 
 /* Errata IPBUNIXTX-39337 */
-- 
2.25.1

